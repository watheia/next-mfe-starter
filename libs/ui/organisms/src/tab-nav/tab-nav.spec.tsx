import { render } from '@testing-library/react';

import TabNav from './tab-nav';

describe('TabNav', () => {
  it('should render successfully', () => {
    const routes = [
      { label: 'A', url: '#a' },
      { label: 'B', url: '#b' },
    ];
    const { container } = render(<TabNav routes={routes} />);
    expect(container).toBeInstanceOf(HTMLElement);
    expect(container).toMatchSnapshot();
  });
});
