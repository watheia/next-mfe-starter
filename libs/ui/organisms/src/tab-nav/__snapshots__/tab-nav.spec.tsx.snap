// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabNav should render successfully 1`] = `
<div>
  <div
    class="MuiBox-root css-1274r9c"
  >
    <div
      class="MuiTabs-root css-1ujnqem-MuiTabs-root"
    >
      <div
        class="MuiTabs-scrollableX MuiTabs-hideScrollbar css-agdkjh-MuiTabs-scrollbarSize"
        style="width: 99px; height: 99px; position: absolute; top: -9999px; overflow: scroll;"
      />
      <div
        class="MuiTabs-scroller MuiTabs-hideScrollbar MuiTabs-scrollableX css-69z67c-MuiTabs-scroller"
        style="margin-bottom: 0px;"
      >
        <div
          aria-label="primary navigation"
          class="MuiTabs-flexContainer css-heg063-MuiTabs-flexContainer"
          role="tablist"
        >
          <a
            aria-selected="false"
            class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary css-1h9z7r5-MuiButtonBase-root-MuiTab-root"
            href="http://localhost:4200/#a"
            role="tab"
            tabindex="-1"
          >
            A
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </a>
          <a
            aria-selected="false"
            class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary css-1h9z7r5-MuiButtonBase-root-MuiTab-root"
            href="http://localhost:4200/#b"
            role="tab"
            tabindex="-1"
          >
            B
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </a>
        </div>
        <span
          class="MuiTabs-indicator css-1aquho2-MuiTabs-indicator"
          style="left: 0px; width: 0px;"
        />
      </div>
    </div>
  </div>
</div>
`;
